### PR TITLE
Add unsupported help for `systemd-resolved`

### PR DESCRIPTION
**Pull request by mdegat01**
## Proposed change

New unsupported OS reason added to supervisor, adding the corresponding help page for users with this issue.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/supervisor/pull/3487
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
The releases of the Supervisor are out of sync with the Home Assistant Core, this PR should probably target the `current` branch for that reason (this can go live in the current documentation without problems anyways).

File: source/more-info/unsupported/systemd_resolved.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Systemd-Resolved"
+description: "More information on why systemd-resolved marks the installation as unsupported."
+---
+
+## The issue
+
+Systemd-resolved is used over DBus to resolve DNS queries made by Home Assistant, Supervisor,
```
Comment (frenck) [inline source/more-info/unsupported/systemd_resolved.markdown]:
```suggestion
systemd-resolved is used over D-Bus to resolve DNS queries made by Home Assistant, Supervisor,
```

File: source/more-info/unsupported/systemd_resolved.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+and Add-Ons. Without it, DNS will not work correctly in your installation.
```
Comment (frenck) [inline source/more-info/unsupported/systemd_resolved.markdown]:
```suggestion
and add-ons. Without it, DNS will not work correctly in your installation.
```

File: source/more-info/unsupported/systemd_resolved.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## The solution
+
+If you see a message about an issue with DBus, [fix that first](https://www.home-assistant.io/more-info/unsupported/dbus#the-solution).
```
Comment (frenck) [inline source/more-info/unsupported/systemd_resolved.markdown]:
```suggestion
If you see a message about an issue with D-Bus, [resolve that first](/more-info/unsupported/dbus#the-solution).
```

**Comment by mdegat01** [comment]
ugh. bad rebase, sorry all

**Review by frenck — APPROVED** [review]
Thanks, @mdegat01 👍

**Pull request merged**